porch/pkg/cmd/server/start.go

```diff
@@ -229,4 +232,5 @@ func (o *PorchServerOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.FunctionRunnerAddress, "function-runner", "", "Address of the function runner gRPC service.")
 	fs.StringVar(&o.DefaultImagePrefix, "default-image-prefix", "gcr.io/kpt-fn/", "Default prefix for unqualified function names")
 	fs.StringVar(&o.CacheDirectory, "cache-directory", "", "Directory where Porch server stores repository and package caches.")
+	fs.DurationVar(&o.RepoSyncFrequency, "repo-sync-frequency", 60, "Frequency in seconds at which registered repositories will be synced.")
```

Comment:
I don't see anywhere to add tests, either. Are you sure this default means 60 seconds? shouldn't it be `60 * time.Second`...is it possible to add a test that at least checks that? Maybe we should add a constraint that the timing can't be less than 30s or something? Up to you, I am OK to merge this once you verify this default value.

Reply:
Thanks for pointing. I was only checking the happy scenario where I provide the flag. 60 was considered as 0 second by the code. I have changed it to `60*time.Second`.  Added a unit test to ensure that this time stays above 30s.